Libplanet.Tests/Action/ActionEvaluatorTest.cs

```diff
@@ -1071,6 +1077,29 @@ public void EvaluatePolicyBlockAction()
             return (addresses, txs);
         }
 
+        [Fact]
+        private async void CheckGenesisHashInAction()
```

Comment:
```suggestion
        private async Task CheckGenesisHashInAction()
```

Generally, `async Task` is more affordable than `async void` [due to controllability](https://docs.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming#avoid-async-void). 
Fortunately, xUnit seems to support `async void` correctly, but I still prefer `async Task` for portability 😅